priv/www/js/tmpl/user.ejs

```diff
@@ -37,20 +37,33 @@
           <th>
             <label>
               <select name="has-password" class="narrow controls-appearance">
-                <option value="password">Password:</option>
+                <% if (user.password_hash.length > 0) { %>
+                <option value="password" selected="selected">Password:</option>
                 <option value="no-password">No password</option>
+                <% } else { %>
+                <option value="no-password" selected="selected">No password</option>
+                <option value="password">Password:</option>
```

Comment:
Was it intentional to reverse the order of the options at the same time? I believe they should remain in the same order, no matter the selected value.

Reply:
We only have two options and the "topmost" will be the selected one. Does it really matter what the order is in this case? It makes sense to me that if the user has no password then the "no password" option would come first.

Reply:
I think it matters because people get used to where things are, to do things quickly.

For instance, I find it very annoying when the same application reverses the order of the "Yes" and "No" buttons in a dialog. This happens with e.g. Firefox when you compare the Windows version to the Unix one. They do this because it matches the platform the application runs on, but as a user using both, it's very confusing.

Reply:
This form is not used nearly as often as Yes and No buttons in system dialogs, especially when the user has no password. We only have two options here and the topmost one will be the one that's currently selected. We do want to preselect the current password state one, which means the other one will come second.

So, unless I'm missing something the order in this case does not change anything for the user.

Reply:
For what it's worth, this confused me the first time I saw it. I was updating a password-less user and I thought that the form was requiring me to supply my own password.

I don't think we need to change the order of the items, so long as the "No password" option is selected correctly. I just changed it so that the "other" option was "below" the current one.

Reply:
I restored the order but for some reason this PR wasn't updated to list my commit. Probably because it's been merged.